CMakePresets.json

```diff
@@ -77,6 +77,23 @@
     }
   ],
   "buildPresets": [
+    {
+      "name": "all",
```

Comment:
sort of a nit but I don't love this name - it really feels like it should build _everything_, but it doesn't and can't - doesn't build tests, doesn't build simulators, doesn't build image hexes, etc. It just builds the application firmwares. Maybe call it `all-application-firmware` or something?